sdk/eventhub/Azure.Messaging.EventHubs/stress/stress-test-resources.json

```diff
@@ -0,0 +1,165 @@
+{
```

Comment:
I would avoid checking this file in and also add a `.gitignore` for it ([example](https://github.com/Azure/azure-sdk-tools/blob/main/tools/stress-cluster/chaos/examples/.gitignore#L1)) since it's an intermediate artifact.